query/src/storages/storage_table.rs

```diff
@@ -118,8 +118,16 @@ pub trait Table: Sync + Send {
         _truncate_plan: TruncateTablePlan,
     ) -> Result<()> {
         Err(ErrorCode::UnImplement(format!(
-            "truncate for local table {} is not implemented",
+            "truncate for table {} is not implemented",
             self.name()
         )))
     }
+
+    async fn truncate_history(
```

Comment:
The function name called `optimize` maybe better?
https://dev.mysql.com/doc/refman/8.0/en/optimize-table.html

Reply:
sure